sklearn/model_selection/_validation.py

```diff
@@ -474,6 +481,14 @@ def _fit_and_predict(estimator, X, y, train, test, verbose, fit_params,
         estimator.fit(X_train, y_train, **fit_params)
     func = getattr(estimator, method)
     predictions = func(X_test)
+    if method in ['decision_function', 'predict_proba', 'predict_log_proba']:
+        true_classes = np.unique(y)
```

Comment:
As mentioned by @jnothman that `return_counts` is not supported for all versions and it's true that only the number of classes is being used, I have thought of another workaround. Will do the change.